lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1402,10 +1442,12 @@ export class UserAgentApplication {
         }
 
         if (accessTokenCacheItem != null) {
+            this.logger.verbose("Evaluating access token found");
             const expired = Number(accessTokenCacheItem.value.expiresIn);
             // If expiration is within offset, it will force renew
             const offset = this.config.system.tokenRenewalOffsetSeconds || 300;
             if (expired && (expired > TimeUtils.now() + offset)) {
+                this.logger.verbose("Token is not expired, renewing token");
```

Comment:
This may be a little confusing. I wouldnt expect a token to renew if it's not expired. Consider specifying why it's renewing, in this case because, expiration is within the offset

Reply:
Sure, makes sense. Changed.